src/reporting/private/dashboard.ts

```diff
@@ -34,6 +34,19 @@ export interface QuicksightCustomResourceProps {
   readonly redshiftProps: RedShiftProps;
 };
 
+export interface NetworkInterfaceCheckCustomResourceProps {
+  readonly networkInterfaces: string;
+  readonly vpcId: string;
+};
+
+export type NetworkInterfaceCheckCustomResourceLambdaProps = {
+  readonly awsAccountId: string;
```

Comment:
Only `networkInterfaces` is sufficient for querying the status of network interfaces.